components/frontend-react/src/API/Downloads.js

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios';
+
+const fetchDownloads = () => {
+  let token = sessionStorage.getItem('token');
```

Comment:
We can use `axios` `interceptors` to inject token for the requests. @agentmilindu what do you think?

Reply:
I was thinking the same. totally agree.